DOCUMENTS/POSTFIX_CONF.txt

```diff
@@ -13,12 +13,19 @@ postconf -m)  Three main.cf variables are involved:
 
 virtual_mailbox_domains = proxy:mysql:/etc/postfix/sql/mysql_virtual_domains_maps.cf
 virtual_alias_maps =
-   proxy:mysql:/etc/postfix/sql/mysql_virtual_alias_maps.cf,
```

Comment:
The reason i remember was that recursive lookups failed.
e.g. some_mail@host aliases to webmaster@host
webmaster@host aliases to my_name@host

In the default settings this lookup failed.
Could you name a setting where this order of handling is needed?